lib/mbaasRequest/mbaasRequest.js

```diff
@@ -175,6 +175,15 @@ function responseHandler(params, cb) {
 
     if (err || (httpResponse && httpResponse.statusCode >= 400)) {
 
+      // MbaaS is unreachable
+      if (err && err.code !== undefined && err.code === "ENOTFOUND" || err.code === "ETIMEDOUT") {
+        log.logger.error("Error : Status Code", err.code, err);
+        err = {
+          httpCode: 500,
+          message: "MBaaS environment is not reachable. Please make the environment available or delete it, and then retry the project delete."
```

Comment:
This error message should be more generic.

Something like, "MBaaS environment is not reachable. Please make the environment available or delete it."

Would be good to also add the Environment ID to the message.